src/tests/ftest/util/rebuild_test_base.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2019 Intel Corporation.
```

Comment:
Shouldn't we just have 2019 here as this is a newly added file?

Reply:
Sure.  I'll make the update.  Thanks.